guix-science/packages/geant4.scm

```diff
@@ -0,0 +1,699 @@
+            "https://cern.ch/geant4-data/datasets/G4RealSurface." version
+            ".tar.gz"))
+      (sha256 (base32 "08382y1258ifs7nap6zaaazvabg72blr0dkqgwk32lrg07hdwm4r")))))
+
+;; G4SAIDDATA
+;;
+(define g4saiddata-2.0
+  (let ((version "2.0"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4SAIDDATA."
+                          version ".tar.gz"))
+      (sha256 (base32 "149fqy801n1pj2g6lcai2ziyvdz8cxdgbfarax6y8wdakgksh9hx")))))
+
+;; G4ABLA
+;;
+(define g4abla-3.1
+  (let ((version "3.1"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4ABLA."
+                          version ".tar.gz"))
+      (sha256 (base32 "1v97q28g1xqwnav0lwzwk7hc3b87yrmbvkgadf4bkwcbnm9b163n")))))
+
+(define g4abla-3.3
+  (let ((version "3.3"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4ABLA."
+                          version ".tar.gz"))
+      (sha256 (base32 "1cd25vckckxkhyx3pvz5swral0rkd4z7akv2dn4fz77fa8r1n10y")))))
+
+;; G4INCL
+;;
+(define g4incl-1.0
+  (let ((version "1.0"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4INCL."
+                          version ".tar.gz"))
+      (sha256 (base32 "0z9nqk125vvf4f19lhgb37jy60jf9zrjqg5zbxbd1wz93a162qbi")))))
+
+(define g4incl-1.2
+  (let ((version "1.2"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4INCL."
+                          version ".tar.gz"))
+      (sha256 (base32 "0zhs1vnrc0vhb1y4q3bscz9y2k9dsnk7ccjg97br42pffdhb307q")))))
+
+;; G4ENSDFSTATE
+;;
+(define g4ensdfstate-2.3
+  (let ((version "2.3"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4ENSDFSTATE."
+            version ".tar.gz"))
+      (sha256 (base32 "00wjir59rrrlk0a12vi8rsnhyya71rdi1kmark9sp487hbhcai4l")))))
+
+;; G4TENDL
+;;
+(define g4tendl-1.4
+  (let ((version "1.4"))
+    (origin
+      (method url-fetch)
+      (uri (string-append "https://cern.ch/geant4-data/datasets/G4TENDL."
+                          version ".tar.gz"))
+      (sha256 (base32 "1q11jxfy5kjwb0jrvwv6dgdxr3h85s6g2bl9kdbfvd681h178wjb")))))
+
+;; -- Geant4 version 11.1.x --
+;;
+;; -- geant4: Geant4 without visualisation -- 
+
+;; (propagated-inputs (list cmake gnu-make gcc-toolchain))
+(define-public geant4-11-1-1
+  (package
+    (name "geant4")
+    (version "11.1.1")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://gitlab.cern.ch/geant4/geant4")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "141fhmh0w8sbp6cckccf3dswn596ds4vgqwc3gz6i53ypyxmv2fw"))))
+    (build-system cmake-build-system)
+    (inputs (list coreutils
+                  gcc-toolchain
```

Comment:
coreutils and gcc-toolchain are already among the default inputs (natively).  Is there a reason for adding them here?